gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,35 @@
+from oslo_config import cfg
+
+OPTS = [
+    cfg.IntOpt(
+        'workers',
+        default=1,
+        help=_('#of workers to create.')
+    ),
+    cfg.StrOpt(
+        'RpcLoadBalancer',
+        default='StickyRoundRobin',
+        choices=['RoundRobin', 'StickyRoundRobin'],
+        help=_('Check sc/core/lb.py for supported rpc lb algos')
+    ),
+    cfg.StrOpt(
+        'modules_dir',
+        default='gbpservice.neutron.nsf.core.test',
+        help=_('Modules path to import ')
+    ),
+    cfg.IntOpt(
+        'evs_polling_interval',
+        default=1,
+        help=_('Polling interval for events in seconds ')
```

Comment:
a) Ok will add more help text
b) 'evs_polling_interval' is unused will remove it.
c) 'polling_interval' -- Is the minimum interval with which periodic task must poll. Events could have different spacing relative to this polling interval.
d) 'report_state_interval' -- This is the interval at which neutron type of *aaS agents should report their state to their corresponding plugins.